packages/react-spinbutton/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@fluentui/react-spinbutton",
+  "version": "0.1.0",
+  "description": "Package to hold SpinButton using button v8",
+  "private": true,
```

Comment:
Probably makes sense for this to remain private until you add files, but once you add the files you'll need to remove this line